packages/pn-pa-webapp/src/pages/components/NewNotification/PreliminaryInformations.tsx

```diff
@@ -54,7 +55,7 @@ const PreliminaryInformations = ({ notification, onConfirm }: Props) => {
     subject: yup.string().required(`${t('subject')} ${tc('required')}`),
     physicalCommunicationType: yup.string().required(),
     paymentMode: yup.string().required(),
-    group: groups.length > 0 ? yup.string().required() : yup.string(),
+    group: (groups.length > 0 && user.hasGroups) ? yup.string().required() : yup.string(),
```

Comment:
we could avoid checking for groups length using hasGroups, if hasGroups is false groups.length would alsa be 0